src/hooks/useCart/useCart.tsx

```diff
@@ -165,55 +136,28 @@ export const useCart = (
   const [cartState, setCartState] = useState<CartState>("DEFAULT");
   const [checkoutResult, setCheckoutResult] = useState<PostTransactionResult>();
   const [error, setError] = useState<Error>();
-  const [quotaResponse, setQuotaResponse] = useState<Quota | null>(null);
-  const [allQuotaResponse, setAllQuotaResponse] = useState<Quota | null>(null);
+  const { quotaResponse, allQuotaResponse, fetchQuota, quotaError } = useQuota(
+    ids,
+    authKey,
+    endpoint
+  );
   const { showAlert } = useContext(AlertModalContext);
   const clearError = useCallback((): void => setError(undefined), []);
 
+  useEffect(() => {
+    if (quotaError) {
+      setError(quotaError);
+    }
+  }, [quotaError]);
+
```

Comment:
Haven't looked closely into the code, what is the reason for mirroring this state?

Reply:
This no longer exists in the code, let me know if this is still an issue.